### PR TITLE
Decrease binding to Entity and derivatives

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: src/Serializers/PropertySerializer.php [hunk 1]
```diff
@@ -72,7 +72,12 @@ private function getSerialized( Property $entity ) {
 			'datatype' => $entity->getDataTypeId(),
 		);
 
-		$this->fingerprintSerializer->addBasicsToSerialization( $entity, $serialization );
```
Comment (Benestar) [inline src/Serializers/PropertySerializer.php]: $entity -> $property

File: src/Serializers/FingerprintSerializer.php [hunk 2]
```diff
@@ -48,15 +47,11 @@ public function addIdToSerialization( Entity $entity, array &$serialization ) {
 		$serialization['id'] = $id->getSerialization();
 	}
 
-	public function addLabelsToSerialization( Entity $entity, array &$serialization ) {
-		$labels = $entity->getFingerprint()->getLabels();
-
+	public function addLabelsToSerialization( TermList $labels, array &$serialization ) {
```
Comment (thiemowmde) [inline src/Serializers/FingerprintSerializer.php]: Many methods here can be private. Not part of this patch, just wanted to point this out.

File: src/Serializers/PropertySerializer.php (hunk 1 above)
Comment (thiemowmde) [inline src/Serializers/PropertySerializer.php]: Correct, but not part of this patch.

File: tests/unit/Serializers/FingerprintSerializerTest.php [hunk 3]
```diff
@@ -312,41 +315,47 @@ public function serializationProvider() {
...
+		$descriptions->setTextForLanguage( 'en', 'A Nyan Cat' );
 
 		$result = array();
 
-		$descriptions = new stdClass();
-		$descriptions->en = array(
-			'language' => 'en',
-			'value' => 'A Nyan Cat'
+		$expectedSerialization = array(
+			'descriptions' => (object)array(
```
Comment (thiemowmde) [inline tests/unit/Serializers/FingerprintSerializerTest.php]: Uh, nice. I like that.

File: src/Serializers/FingerprintSerializer.php (hunk 2 above)
Comment (JeroenDeDauw) [inline src/Serializers/FingerprintSerializer.php]: Yeah... I'm quite unhappy about what that code looked like, this commit only fixing some of it. Too bad it got through review somehow

File: tests/unit/Serializers/FingerprintSerializerTest.php (hunk 3 above)
Comment (Benestar) [inline tests/unit/Serializers/FingerprintSerializerTest.php]: `use stdClass;` can be removed then, right?

Comment (thiemowmde) [inline tests/unit/Serializers/FingerprintSerializerTest.php]: Already done in #140.

Comment (Benestar) [inline tests/unit/Serializers/FingerprintSerializerTest.php]: :+1:
